python/lsst/meas/base/plugins.py

```diff
@@ -439,12 +439,14 @@ def __init__(self, config, name, schema, metadata):
                 "calibration factor at the location of the src.")
 
     def measure(self, measRecord, exposure, center):
-
         photoCalib = exposure.getPhotoCalib()
-        calib = photoCalib.getLocalCalibration(center)
+        if photoCalib is None:
+            calib = np.nan
+            calibErr = np.nan
```

Comment:
1) Is it worth having a debug log here?  Could go either way.
2) There is also a field `base_LocalPhotoCalib_flag` that was set when this failed (and similarly `base_LocalWcs_flag`). I think we need to set these, but I'm not sure off the top of my head how to get to these field names ...

A-ha!  https://github.com/lsst/meas_base/blob/2f1216ac18b67d915507f5ef145f1b8649c3ee01/python/lsst/meas/base/wrappers.py#L547 shows that you can do `measRecord.set(self._failKey, True)` because this is a "generic plugin".